app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @challenge = @project.challenge
+  end
```

Comment:
We typically only want to send a single instance variable to the view, so just sending `@project` is sufficient and we can get the challenge from that when needed.